bootiso

```diff
@@ -1742,14 +1882,12 @@ main() {
 	assignInternalVariables
 	checkFlagMatrix
 	checkPackages
-	if [ "$expectingISOFile" == 'true' ]; then
```

Comment:
Yes this is intended : if the ISO is not OK, I don't want the user to have typed his sudo credentials for nothing :-)

Reply:
You ok with moving the hash check to after checkSudo? Because the hash will be checked twice if it's in front. (Once as normal user, once as root)
So you get duplicate messages as output, and hashing a large file could be a 'expensive' operation.

Reply:
@SibrenVasse OK, please do!